tools/truncate-silence.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+tools_path=$(dirname "$0")
+
+source "$tools_path/check-ffmpeg.sh"
+if ! check_ffmpeg; then
+    exit 1
+fi
+
+usage() {
+    echo "Usage: $0 [OPTION] [ARGS]" 1>&2;
+    echo "Truncates silence from the start or the end of an audio file"
+    echo "    --check-codecs    Checks if the encoders used by the HackSoundServer are installed." 1>&2
```

Comment:
If I understand correctly, the codecs are not needed by this script, so I think it's OK to remove this option.